rules/__tests__/valid-describe.test.js

```diff
@@ -45,7 +45,8 @@ ruleTester.run('valid-describe', rule, {
       code: 'describe(() => {})',
       errors: [
         {
-          message: 'First argument must be name',
+          message:
```

Comment:
Look at the code - this should keep the old message. Cab we differentiate between missing first arg/wrong type and a variable?

Reply:
I don't think I follow. Could you explain what exactly you're looking for?

Reply:
I think the point is that the new message suggests that the mistake was to pass a function instead of a string, when the mistake is really that the name is just missing before the function. The old message was neutral in this regard.
And I guess @SimenB's suggestion (second sentence) was to print something like "First argument must be name" in this case because we can infer that the user probably just forgot the name in before the function, and only if both args are there but the first is not a string literal print the new one :)

Reply:
Yes, that's what I meant. Thanks! 😀